spec/rails_helper.rb

```diff
@@ -1,35 +1,35 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'simplecov'
+SimpleCov.start 'rails' do
+  add_filter 'app/channels/application_cable/'
+  add_filter 'app/jobs/'
+  add_filter 'app/mailers/'
+  add_filter 'app/models/application_record.rb'
+
+  
+end
```

Comment:
Please, move it to spec/support/config/*

and you can require it this way
Dir[Rails.root.join('spec', 'support', '**', '*.rb')].sort.each { |file| require file }